### ISSUE TITLE
Social Preview image (aka Twitter Card Summary) does not always show proper fonts and does not show images added in markdown 

### ISSUE THREAD

**Issue by cointastical** [stackernews/stacker.news#99]
The Social Preview image (used when someone Tweets a link for an SN post in Twitter) does not render an image (if in a Description post, added using markdown), nor does the proper font always get used in that preview image.

To see what SN prepares for the preview, use the Twitter Card Validator.
http://cards-dev.twitter.com/validator

On a related note, is there any reason a Socail Preview / Twitter Card Summary couldn't  be created when the URL is for a Comment or a Comment Reply as well?
Right now there is no image (though the Twitter Card Validator doesn't show any error).

**Comment by cointastical** [comment]
For further details on this, see the discussion on [SN Comment Reply, Item #10558](https://stacker.news/items/10558)

**Comment by huumn** [comment]
This appears to be functional now: https://stacker.news/api/capture/items/10459. For the font load I had a CORS issue given the origin to generate the screenshot was localhost and I had switched to a CDN and set a restrictive CORS. In my tests the markdown images also load ... I'm not sure if those were broken or it's an intermittent issue.

The twitter validator doesn't wait for the screenshot to be captured so it's not a great simulation of how twitter cards work on twitter.

Closing and will watch the issue.
